tasks/flush_repo.py

```diff
@@ -151,10 +155,29 @@ def _delete_label_analysis(self, db_session: Session, commit_ids, repoid: int):
 
     @sentry_sdk.trace
     def _delete_commits(self, db_session: Session, repoid: int) -> int:
-        deleted_commits = db_session.query(Commit).filter_by(repoid=repoid).delete()
+        commits_to_delete = db_session.query(Commit).filter_by(repoid=repoid).all()
+        commit_ids_to_delete = [commit.commitid for commit in commits_to_delete]
+
+        # Reset parallel session id redis key for each commit
+        for commit in commits_to_delete:
```

Comment:
even though there is a TTL, I think we should still reset redis keys explicitly here. Otherwise if a user tries to do uploads for a commit they just erased, the session IDs will be higher than they need to.